extensions/wikia/WikiFactory/Dumps/maintenance/DumpsOnDemandCron.php

```diff
@@ -49,13 +45,19 @@ public function execute() {
 
         $this->output( "INFO: Creating dumps for Wikia #{$sWikiaId}.\n" );
 
-        $sCommand = sprintf( 'SERVER_ID=177 php %s/extensions/wikia/WikiFactory/Dumps/runBackups.php --conf %s --id=%d --both --tmp --s3', $IP, $wgWikiaLocalSettingsPath, $sWikiaId );
-
-        wfShellExec( $sCommand, $iStatus );
+        $sCommand = sprintf( 'SERVER_ID=177 php %s/extensions/wikia/WikiFactory/Dumps/runBackups.php --conf %s --id=%d --both --tmp --s3 2>&1', $IP, $wgWikiaLocalSettingsPath, $sWikiaId );
```

Comment:
It's a [community wiki[(http://community.wikia.com/wiki/Special:WikiFactory/177) - I'll use a const here instead (I believe we have one).